MoreLinq/CountDown.cs

```diff
@@ -65,15 +65,11 @@ static partial class MoreEnumerable
 
             IEnumerable<TResult> IterateList(IListLike<T> list)
             {
-                var countdown = Math.Min(count, list.Count);
+                var count = list.Count;
+                var countdown = Math.Min(count, count);
```

Comment:
This is effectively a no-op, since both parameters are the same.

Reply:
Yep, looks like I made a silly mistake and the [tests broke](https://ci.appveyor.com/project/raboof/morelinq/builds/46021323/job/c7ad76k2b4qypcv0#L448) in the CI build. On the plus side, good things tests caught it. 😅